src/main/java/org/prgrms/application/domain/voucher/VoucherType.java

```diff
@@ -0,0 +1,16 @@
+package org.prgrms.application.domain.voucher;
+
+import java.util.Arrays;
+
+public enum VoucherType {
+    FIXED,
+    PERCENT;
+
+    public static VoucherType findBySelection(String selection) {
+        return Arrays.stream(values())
+                .filter(s -> s.name().equals(selection.toUpperCase()))
```

Comment:
```suggestion
                .filter(s -> s.name().equalsIgnoreCase(selection))
```
p3; 
별건 아니지만 다음과 같이 바꿔도 동일하게 동작할 것 같아요~

Reply:
오옹 새로운 문법을 알아갑니다!